tanguyantoine/react-native-music-control
# Add custom notification icon options for Android

#### What's this PR do?
This feature allows library users to supply their own custom icons to be used by the notification on Android. This was technically already possible, but the custom icon lookup string used previously required the asset to contain `-` which is not supported by Android's resource system.

The PR also adds a `setCustomNotificationIcon` function that allows the developer to use a custom resource name.

#### Which issue(s) is it related to?
Not related to an issue, but to a feature I needed.

#### How to test:
To test the PR, you have to create a drawable resource in your Android app. In the resource, you should add an icon named `music_control_icon.png`. You can just add it top level to drawable or use multiple resolutions created by various tools or Android Studio's notification icon UI. Now the notification should be shown with a custom icon.

Now add another icon with a different name, for example `my_custom_icon.png`.
In your code add
```javascript
  MusicControl.setCustomNotificationIcon('my_custom_icon');
```
Now the notification should be shown with a custom icon.
